src/markitdown/_markitdown.py

```diff
@@ -857,6 +868,31 @@ def _convert_chart_to_markdown(self, chart):
         separator = "|" + "|".join(["---"] * len(data[0])) + "|"
         return md + "\n".join([header, separator] + markdown_table[1:])
 
+    def _get_llm_description(self, image_blob, client, model, prompt=None):
+        if prompt is None or prompt.strip() == "":
+            prompt = "Write a caption for this image."
+        content_type = "image/jpeg"
```

Comment:
Does ```content_type = "image/jpeg"``` assumes that the file is only jpeg ?

It can be anything, Including below too, Yes?

JPEG Images: "image/jpeg"
PNG Images: "image/png"
GIF Images: "image/gif"

Reply:
Found out the ‘image’ object has content_type. Using it directly for the API call. TY!